### DISCUSSION TITLE
Filter: automatically adjust delimiter size

### DISCUSSION THREAD

**Discussion by GuillaumeDehaene** [quarto-dev/quarto-cli#7327] [Show and Tell]
### Description

Hello everyone,

I've spent the weekend hacking together a filter to automatically adjust delimiter size in math
https://github.com/GuillaumeDehaene/adjust_delimiter_size/tree/0.1.0

It's quite simple: by default, latex delimiters do not adjust to match the size of their contents, but they do in typst.
My extension fixes that:

$$
( \frac{1}{n} \sum_{i=1}^n x_i )^2 \rightarrow \left( \frac{1}{n} \sum_{i=1}^n x_i \right)^2
$$

Everything is automatic.

I'd be very happy to get feedback on it.

Cheers,
Guillaume

**Comment by mcanouil** [comment]
Great job!

I believe it would be clearer if you publish the rendered example to GitHub Pages, as currently it is unclear from the Readme what your extension actually does.

Possibly a with/without table in the readme could help show what you propose.

(This also applies to your other extension)
